en/docs/setup/installation-prerequisites.md

```diff
@@ -41,28 +35,31 @@ development platforms before starting the installation.
+<th><p>Operating Systems/ Databases/ User Stores</p></th>
 <td><div class="content-wrapper">
 <ul>
 <li>WSO2 Identity Server requires an Oracle JDK 8 compliant JDK. This will run on most common platforms that <strong>support Java 8</strong> .</li>
-<li>All WSO2 Carbon-based products are generally compatible with most common DBMSs. The embedded H2 database is suitable for development and testing. For enterprise production environments we recommend an industry-standard RDBMS such as Oracle, PostgreSQL, MySQL, MS SQL, etc. For more information, see <a href="https://docs.wso2.com/display/ADMIN44x/Working+with+Databases">Working with Databases</a> .</li>
+<li>All WSO2 Carbon-based products are generally compatible with most common DBMSs. The embedded H2 database is suitable for development and testing. For enterprise production environments we recommend an industry-standard RDBMS such as Oracle, PostgreSQL, MySQL, MS SQL, etc. For more information, see <a href="../../admin-guide/working-with-databases">Working with Databases</a> .</li>
 <li>WSO2 Identity Server supports using any of the following as a user store :
 <ul>
 <li>RDBMS</li>
 <li>An LDAP such as OpenLDAP</li>
 <li>Active Directory</li>
-<li><a href="_Configuring_User_Stores_">Custom user stores</a></li>
+<li><a href="../../using-wso2-identity-server/configuring-user-stores">Custom user stores</a></li>
 </ul></li>
 </ul>
-!!! note
-    <p>Note</p>
-    <ul>
-    <li>WSO2 Identity Server is shipped with a default embedded Apache DS that is suitable for development purposes. WSO2 does not recommend using Apache DS in a production environment due to scalability issues that exist with Apache DS.</li>
-    <li>WSO2 does not recommend using the H2 database as a user store in production environments. However, you can use the H2 database for development purposes if necessary.</li>
-    </ul>
+<div class="admonition note">
+<p class="admonition-title">Note</p>
+<p>
 <ul>
-<li>For environments that WSO2 products are tested with, see <a href="https://docs.wso2.com/display/TestedPlatforms/Environments+Tested+with+WSO2+Products">Environments Tested with WSO2 Products</a> .</li>
-<li>If you have difficulty setting up any WSO2 product in a specific platform or database, <a href="https://wso2.com/contact/">contact WSO2</a> .</li>
+<li>WSO2 Identity Server is shipped with a default embedded Apache DS that is suitable for development purposes. WSO2 does not recommend using Apache DS in a production environment due to scalability issues that exist with Apache DS.</li>
+<li>WSO2 does not recommend using the H2 database as a user store in production environments. However, you can use the H2 database for development purposes if necessary.</li>
+</ul>
+</p>
+</div>
+<ul> 
+<li>For environments that WSO2 products are tested with, see <a href="../../admin-guide/environments-tested-with-wso2-products">Environments Tested with WSO2 Products</a> .</li>
```

Comment:
change this link path to "administer". Please check **all** relevant places and fix according to the new folder structure